_pages/robots.html

```diff
@@ -47,14 +47,14 @@ <h5>2015</h5>
             <p>Our 2015 robot, which we built for the <a href="https://www.youtube.com/watch?v=mAN1B7oKDXE">Recycle Rush</a> challenge, had the ability to stack totes quickly and effectively and manipulate recycling cans and trash, even in autonomous mode.</p>
         </div>
         <div onclick="location.href='robot/2014.html'" style="cursor:pointer;">
-            <a href="robot/2014.html"><img src="/images/robots/2014.png">
+            <img src="/images/robots/2014.png">
             <h5>2014</h5>
             <p>Our 2014 robot, which was designed for the <a href="https://www.youtube.com/watch?v=oxp4dkMQ1Vo">FIRST Aerial Assist</a> challenge, was able to catapult large balls with 100% consistency.</p>
-        </div>
-        <div>
+        </div>    
```

Comment:
Nit: trailing whitespace

Reply:
I would recommend configuring your IDE to strip trailing whitespace automatically